test/integration/acceptance/CMakeLists.txt

```diff
@@ -83,7 +83,11 @@ target_link_libraries(tx_heavy_data
     acceptance_fixture
     )
 
-addtest(get_account_assets_test get_account_assets_test.cpp)
+addtest(get_account_assets_test
+    "get_account_assets_test.cpp;\
+query_permission_test_ast.cpp;\
```

Comment:
D: maybe link as separate target?

Reply:
what exactly would you like to separate?

Reply:
targets that follow get_account_assets_test.cpp

Reply:
targets that follow get_account_assets_test.cpp